Source/Scene/Scene.js

```diff
@@ -2562,6 +2562,9 @@ define([
 
         us.update(frameState);
 
+        var backgroundColor = defaultValue(scene.backgroundColor, Color.BLACK);
+        us.updateBackgroundColor(backgroundColor);
```

Comment:
Since `UniformState.prototype.updateBackgroundColor` seems somewhat specific to add, maybe `backgroundColor` could be added to `FrameState` instead and updated in that way.